docs/examples/langchain/chroma_pdf_qa.py

```diff
@@ -5,6 +5,7 @@
 
```

Comment:
Some material style issues to fix

![image](https://github.com/holoviz-topics/panel-chat-examples/assets/42288570/b571f198-336b-4072-9081-56d17f4ff683)

Reply:
Yes. The two one is because the "hidden" symbol is not centered vertically. Its not too bad. I believe its much worse for Bootstrap and Fast. The second one is because the button is using Bokeh/ Bootstrap design. Not something that looks like a Material Button group.

Reply:
The hidden symbol I think may be a bokeh issue.

I think the Material button style can be submitted as Panel issue.